scripts/hyper-runner.sh

```diff
@@ -105,6 +107,10 @@ if [[ -z "$STORE_URI" ]]; then
   log "--store_uri is a required argument";
   printUsage;
 fi
+if [[ -z "$UI_URI" ]]; then
```

Comment:
shall we relax the check here? so that UI_URI is optional?

Reply:
should we assign a default or just ignore if missing?

Reply:
Added default 👍